specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/preview/2023-03-01-preview/iotdps.json

```diff
@@ -0,0 +1,2631 @@
+    },
+    "ProvisioningServiceDescriptionListResult": {
+      "type": "object",
+      "description": "List of provisioning service descriptions.",
+      "properties": {
+        "value": {
+          "description": "List of provisioning service descriptions.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ProvisioningServiceDescription"
+          }
+        },
+        "nextLink": {
+          "description": "the next link",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "VerificationCodeResponse": {
+      "type": "object",
+      "description": "Description of the response of the verification code.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of certificate.",
+          "readOnly": true
+        },
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Request etag."
+        },
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource identifier."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource type."
+        },
+        "properties": {
+          "type": "object",
+          "properties": {
+            "verificationCode": {
+              "type": "string",
+              "description": "Verification code."
+            },
+            "subject": {
+              "type": "string",
+              "description": "Certificate subject."
+            },
+            "expiry": {
+              "type": "string",
+              "description": "Code expiry."
+            },
+            "thumbprint": {
+              "type": "string",
+              "description": "Certificate thumbprint."
+            },
+            "isVerified": {
+              "type": "boolean",
+              "description": "Indicate if the certificate is verified by owner of private key."
+            },
+            "certificate": {
+              "description": "base-64 representation of X509 certificate .cer file or just .pem file content.",
+              "type": "string",
+              "format": "byte"
+            },
+            "created": {
+              "type": "string",
+              "description": "Certificate created time."
+            },
+            "updated": {
+              "type": "string",
+              "description": "Certificate updated time."
+            }
+          }
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "VerificationCodeRequest": {
+      "type": "object",
+      "description": "The JSON-serialized leaf certificate",
+      "properties": {
+        "certificate": {
+          "description": "base-64 representation of X509 certificate .cer file or just .pem file content.",
+          "type": "string"
+        }
+      }
+    },
+    "ArmIdentity": {
```

Comment:
>ArmIdentity [](http://example.com/codeflow?start=5&length=11)

Theres a common type for managed identities. Please use that.

Reply:
[https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/managedidentity.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/managedidentity.json)

Reply:
Do you mean that our ArmIdentity and ArmUserIdentity definitions need to follow the same format as the managedidentity.json sent in the link? Does the naming also need to be changed to "ManagedServiceIdentity" etc?